spanner/database_leader_placement_samples.rb

```diff
@@ -0,0 +1,240 @@
+# Copyright 2021 Google, Inc
```

Comment:
Use `Google LLC` (rather than Google, Inc) in the copyright lines for all new files. (There are two such files in this PR.)

Reply:
done